source/_includes/asides/component_navigation.html

```diff
@@ -73,5 +73,36 @@
     <a href="https://github.com/home-assistant/core/tree/dev/homeassistant/components/{{ domain }}" target="_blank"><i class="icon-github"></i> View source on GitHub</a><br />
     <a href="https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+{{ domain }}%22" target="_blank"><i class="icon-bug"></i> View known issues</a>
   </div>
-  {%- endif -%}
+  {%- endif -%}  
+</section>
+
+<section class="aside-module grid__item one-whole lap-one-half">
+  <div class='section'>
+    <h1 class="title delta">Codeowners</h1>
+    {% if page.ha_codeowners %}
+        {% assign ha_project = false %}
+        {% for codeowner in page.ha_codeowners %}
+            {% if codeowner contains "@home-assistant/" %}
+                {% assign ha_project = true %}
+            {% endif %}
+        {% endfor %}
+        
+        {% if ha_project %}
+            This integration is being maintained by the Home Assistant project.
+        {% else %}
+          We are incredibly grateful to the following contributors who currently maintain this integration:<br />
```

Comment:
```suggestion
          We are incredibly grateful to the following contributors who currently maintain this integration:
```
Should do a single here to match the section above it.

Reply:
It looks really weird if we do that... As in, the list is almost embedded in the above text at that point.

Reply:
Meh... yeah that did not look any better.
For it to be natural with the other sections, it need more defined sections and not a section within another section, like this:
```diff
--- a/source/_includes/asides/component_navigation.html
+++ b/source/_includes/asides/component_navigation.html
@@ -77,7 +77,6 @@
 </section>
 
 <section class="aside-module grid__item one-whole lap-one-half">
-  <div class='section'>
     <h1 class="title delta">Codeowners</h1>
     {% if page.ha_codeowners %}
         {% assign ha_project = false %}
@@ -88,10 +87,13 @@
         {% endfor %}
         
         {% if ha_project %}
+          <div class='section'>
             This integration is being maintained by the Home Assistant project.
+          </div>
         {% else %}
-          We are incredibly grateful to the following contributors who currently maintain this integration:<br />
-          <br />
+        <div class='section'>
+          We are incredibly grateful to the following contributors who currently maintain this integration:
+          </div>
           <div class="section">
             {%- for codeowner in page.ha_codeowners -%}
             {%- assign clean_codeowner = codeowner | replace: "@", "" -%}
@@ -100,9 +102,10 @@
           </div>
         {% endif %}
     {% else %}
-        This integration is community maintained.<br />
-        If you are a developer and would like to help, feel free to contribute!
+        <div class='section'>
+          This integration is community maintained.<br />
+          If you are a developer and would like to help, feel free to contribute!
+        </div>
     {% endif %}
-  </div>
 </section>
 ```

Reply:
Thanks! much better 👍